net/openvpn3/Portfile

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+name                openvpn3
+github.setup        OpenVPN openvpn3 3.6.3 release/
```

Comment:
The `name` field matches the project name field in `github.setup`, so I don't think you need to redundantly specify `name`